proto/viam/app/v1/app.proto

```diff
@@ -1353,12 +1356,25 @@ message ListOrganizationsByUserRequest {
 message OrgDetails {
   string org_id = 1;
   string org_name = 2;
+  optional string org_cid = 3;
+  optional string public_namespace = 4;
 }
 
 message ListOrganizationsByUserResponse {
   repeated OrgDetails orgs = 1;
 }
 
+message SearchOrganizationsRequest {
+  optional string org_id = 1;
+  optional string org_name = 2;
+  optional string cid = 3;
+  optional string public_namespace = 4;
+}
```

Comment:
[self] didn't use `oneof` per our [eng guidelines](https://viam.atlassian.net/wiki/spaces/ENG/pages/262371/Viam+Engineering+Guidelines)
>Avoid oneof except for enums, since non-scalar oneofs are hard to work with in most protoc generations